### DISCUSSION TITLE
assign an Issue

### DISCUSSION THREAD

**Discussion by SIBI-MS** [kovidgoyal/kitty-fosshack2024#6] [Show and tell]
i have experience in python, djnago and also LLM frameworks like langchain, llamaindex etc..i want to work in an issue

**Comment by kovidgoyal** [comment]
On Mon, Jul 22, 2024 at 07:26:24AM -0700, SIBI M. S. wrote:
> i have experience in python, djnago and also LLM frameworks like langchain, llamaindex etc..i want to work in an issue

Look at the open, un assigned issues and pick one you like.
